src/asm/fstack.cpp

```diff
@@ -240,7 +240,11 @@ bool yywrap(void)
 
 		// If this is a FOR, update the symbol value
 		if (contextStack->forName && fileInfo->iters[0] <= contextStack->nbReptIters) {
-			contextStack->forValue += contextStack->forStep;
+			// Avoid arithmetic overflow runtime error
```

Comment:
I want to ~~die~~ stop using these wretched languages, but ok

Reply:
Does C++20 have a better alternative?